NEWS

```diff
@@ -75,6 +75,10 @@ PHP                                                                        NEWS
   . Fixed bug GH-15628 (php_stream_memory_get_buffer() not zero-terminated).
     (cmb)
 
+- MBString:
+  . Fixed bug GH-15824 mb_detect_encoding(): Argument $encodings contains
+    invalid encoding "UTF8". (Yuya Hamada)
```

Comment:
Bug description should be between ():

```suggestion
  . Fixed bug GH-15824 (mb_detect_encoding(): Argument $encodings contains
    invalid encoding "UTF8"). (Yuya Hamada)
```

Reply:
Thanks! Added.

Reply:
Note that this is important for web-php/news2html which is used to create the [changelog](https://www.php.net/ChangeLog-8.php) from NEWS.